coordinator/src/main/scala/com/socrata/datacoordinator/primary/MigrateSchema.scala

```diff
@@ -37,5 +37,6 @@ object MigrateSchema extends App {
 
     SchemaMigrator(databaseTree, operation, numChanges)
   }
-  private lazy val databaseTree = "com.socrata.coordinator.common.database"
+
+  private lazy val databaseTree = s"${com.socrata.datacoordinator.service.Main.configRoot}.database"
```

Comment:
Oh. I see.  It is the fully qualified variable name including package.  That's fine, but slightly confusing.  I think it would be better if you imported Main and just did Main.configRoot, or something similar.